api/user_profile.go

```diff
@@ -30,18 +35,53 @@ func (svc *UserProfileService) Get() (response UserProfileResponse, err error) {
 }
 
 type UserProfileResponse struct {
-	Data []struct {
-		Username   string `json:"username"`
-		OrgAccount bool   `json:"orgAccount"`
-		URL        string `json:"url"`
-		OrgAdmin   bool   `json:"orgAdmin"`
-		OrgUser    bool   `json:"orgUser"`
-		Accounts   []struct {
-			Admin       bool   `json:"admin"`
-			AccountName string `json:"accountName"`
-			CustGUID    string `json:"custGuid"`
-			UserGUID    string `json:"userGuid"`
-			UserEnabled int    `json:"userEnabled"`
-		} `json:"accounts"`
-	} `json:"data"`
+	Data []UserProfile `json:"data"`
+}
+
+type UserProfile struct {
+	Username   string    `json:"username"`
+	OrgAccount bool      `json:"orgAccount"`
+	URL        string    `json:"url"`
+	OrgAdmin   bool      `json:"orgAdmin"`
+	OrgUser    bool      `json:"orgUser"`
+	Accounts   []Account `json:"accounts"`
+}
+
+func (p *UserProfile) OrgAccountName() string {
+	// TODO @afiune should we handle other datacenters?
```

Comment:
I am trying to get access to an account deployed in our Frankfurt datacenter 

![tenor-54643987](https://user-images.githubusercontent.com/5712253/119717195-39ee8d80-be23-11eb-80d0-f9591f5edc2c.gif)

Reply:
![tenor-168906724](https://user-images.githubusercontent.com/5712253/119903605-2a944080-bf06-11eb-8744-b22ef6f880b1.gif)